### ISSUE TITLE
[Question] Tests decisions

### ISSUE THREAD

**Issue by malparty** [wildan3105/google-scraper#39]
When opening the GitHub project, I was very happy to see GitHub actions running tests for both the backend and frontend apps.

However, I was surprised to see the areas chosen to be covered by tests. Could you share more about why the tests have been implemented onto these specific areas: toast, notFound, healthCheck, and LogoutButton?

**Comment by wildan3105** [comment]
Hey, thanks for the comments. 

To be perfectly honest with you, I only have less than 12 hours to create all of those tests (since I was focusing on the app's functionalities, deployment, and (manual) regression test). Hence the decision was to create setup test (for backend -> healthcheck) and several easy-to-test components for frontend. If I had more time, I would definitely cover the essential and critical app flows like the one you've been creating issues for.

So in conclusion, I am happy to take the next few days to cover those areas. Let me know if you need clarification.

**Comment by malparty** [comment]
As mentioned in the other issue, you don't need to handle more code changes at this stage. 

Thanks for your insights :pray:

**Comment by wildan3105** [comment]
Very well, noted on that. Thanks a lot for your insight.
